src/Popover/Popover.js

```diff
@@ -1,5 +1,5 @@
 import classnames from 'classnames';
-import {POPOVER_ALIGNMENTS} from '../utils/constants';
+import { POPOVER_ALIGNMENTS } from '../utils/constants';
```

Comment:
unnecessary whitespace change. If you prefer this style, please open a separate PR, with a lint rule enforcing this style

```
object-curly-spacing: [2, always]
```

Reply:
The shift again happened from some VS Code setting on a TypeScript extension that I removed. Will look into why it's being applied or will add the lint rule.